gnss/ubloxrover.cpp

```diff
@@ -285,3 +288,9 @@ void UbloxRover::saveOnShutdown()
     // UPD-SOS-BACKUP message
     mUblox.ubloxUpdSos(0);
 }
+
+void UbloxRover::gnssToCAN(const ubx_nav_pvt &pvt) {
```

Comment:
This part will be necessary to convert from ubx-specific struct to QVariant and UbloxRover becomes aware of "CAN" through this. I would rather see UbloxRover independent of CAN and move this part to the CANopen implementation by 1. emitting ubx_nav_pvt like Ublox::rxNavPvt does here and 2. doing the CAN-specific conversion to QVariant in CANOpenMovementController.